tests/test_transpile.py

```diff
@@ -322,7 +322,6 @@ def test_alter(self):
             "ALTER TABLE integers ADD k INTEGER",
             "ALTER TABLE integers ADD COLUMN k INT",
         )
-        self.validate("ALTER TABLE integers DROP k", "ALTER TABLE integers DROP COLUMN k")
```

Comment:
This test was removed because `DROP k` now transpiles back to itself, since it's parsed as a command.